docs/ldml/tr35-collation.md

```diff
@@ -44,6 +44,7 @@ The LDML specification is divided into the following parts:
 *   Part 6: [Supplemental](tr35-info.md#Contents) (supplemental data)
 *   Part 7: [Keyboards](tr35-keyboards.md#Contents) (keyboard mappings)
 *   Part 8: [Person Names](tr35-personNames.md#Contents) (person names)
+*   Part 9: [Message Format](tr35-messageFormat.md#Contents) (message format)
```

Comment:
Generally we refer to it as MessageFormat without a space

```suggestion
Part 9: [MessageFormat](tr35-messageFormat.md#Contents) (message format)
```

Reply:
```suggestion
*   Part 9: [MessageFormat](tr35-messageFormat.md#Contents) (message format)
```

Reply:
done